docs/DeveloperGuide.md

```diff
@@ -1,29 +1,134 @@
+
+
+Step 3: Then once it is successfully written to the `currencies.txt` file, then the previously created `Currency` is added both to the `currencyList` and to the `personalCurrencyList`. We need to keep track of both so it’s possible to remove that same currency from that `currencies.txt` file later.
+
+Step 4: Let’s say the user now wants to remove the previously created personal currency from the list of available currencies. It can type in the command `remove` and get prompted with this : 
+OK, which currency would you like to remove? Enter the abbreviation of that currency: `btc`
 
+Step 5: Now we have the abbreviation of the currency and we will add `‘!’` in front of it and create a new temporary currency object based on that with the `findCurrencyByAbbrName` method of the `CurrencyList` class and then check if the currencyList contains that currency. If so, then the `deleteFromCurrencies` method of the `NewCurrencyFileWorkings` class is called, which removes that currency from both of the currency lists and rewrites the `currencies.txt` file without that personal currency.
 
 ## Product scope
 ### Target user profile
 
-{Describe the target user profile}
+ Someone who works with many types of currencies, is interested in the forex market, or is interested in having a completely online wallet for tracking the money in the account.
 
 ### Value proposition
 
-{Describe the value proposition: what problem does it solve?}
+User who has a wallet which can have multiple currencies and can change them from one currency to another, essentially a bank account manager, where user can also withdraw and add money to the account. If they are trying to withdraw more money than they have in one currency we give them the option of exchange some from another currency (e.g trying to withdraw 100 SGD, but only have 80 SGD, we would tell them they can convert 14.34 EUR to match the remaining 20 SGD). They can give out requests to convert either one, select amount of currencies or all of the currencies they have to one specific currency. 
 
 ## User Stories
 
 |Version| As a ... | I want to ... | So that I can ...|
 |--------|----------|---------------|------------------|
-|v1.0|new user|see usage instructions|refer to them when I forget how to use the application|
-|v2.0|user|find a to-do item by name|locate a to-do without having to go through the entire list|
+|v1.0|New User|Create a wallet| can access my wallet and see its total value|
```

Comment:
Perhaps the users in the user stories could be more specific? You could perhaps expand on who(like a bank manager, or maybe working adults, or someone else) will specifically be using your product and try to appeal to their more niche tastes.